release.md

```diff
@@ -0,0 +1,298 @@
++ Configure the number of block **confirmations** to wait between deployments. This is helpful when deploying to Infura because their load balancer sometimes executes back-to-back transactions out of sequence and  noncing can go awry.
++ Specify how many **blocks to wait** before timing out a pending deployment. Web3 hardcodes this value at 50 blocks which can be a problem if you're trying to deploy large contracts at the lower end of the gas price range.
++ A deployer interface that works seamlessly with ES6 **async/await** syntax. (Also backward compatible with Truffle V4's then-able pattern.)
+
+⚠️  **Important** ⚠️  If you're using `truffle-hdwallet-provider` with Truffle V5 you **must** install the Web3 1.0 enabled version:
+```shell
+$ npm install --save truffle-hdwallet-provider@web3-one
+```
+
+**Example network config**
+```javascript
+ropsten: {
+  provider: () => new HDWalletProvider(mnemonic, `https://ropsten.infura.io`),
+  network_id: 3,
+  gas: 5500000,           // Default gas to send per transaction
+  gasPrice: 10000000000,  // 10 gwei (default: 20 gwei)
+  confirmations: 2,       // # of confs to wait between deployments. (default: 0)
+  timeoutBlocks: 200,     // # of blocks before a deployment times out  (minimum/default: 50)
+  skipDryRun: true        // Skip dry run before migrations? (default: false for public nets )
+},
+```
+
+**Example Migration using async / await**
+```javascript
+const One = artifacts.require("One");
+const Two = artifacts.require("Two");
+
+module.exports = async function(deployer) {
+  await deployer.deploy(One);
+
+  const one = await One.deployed();
+  const value = await one.value();
+
+  await deployer.deploy(Two, value);
+};
+```
+**Deploying to Rinkeby...**
+
+
+![migrate-rinkeby](https://user-images.githubusercontent.com/7332026/43867960-3499922c-9b20-11e8-8553-589308a6cd61.gif)
+
+**More migrations improvements are coming soon...**.
++ cag from Gnosis has written a really nice addition to the Migrations module that will automatically deploy contract dependencies you've installed with `npm` along with your own contracts.
++ Under the hood, the `migrations` command is now completely evented and managed by a reporter module. Those hooks will be exposed so anyone can write a UI for it and hopefully you'll be plugging into ever more sophisticated deployment script managers soon. Work on the default UI to make it more interactive and colorful is ongoing.
+
+## Even More!
+
+### truffle-console now supports async/await
+
+A small but decisive improvement that should make the console much easier to use.
+**Example**
+```shell
+truffle(development)> let instance = await MetaCoin.deployed()
+truffle(development)> let accounts = await web3.eth.getAccounts()
+truffle(development)> let balance = await instance.getBalanceInEth(accounts[0])
+truffle(development)> balance.toNumber()
+20000
+truffle(development)>
+```
```

Comment:
We'll likely need:
- Contributing section / Call for feedback to get everything perfect for the full release
- Thank you to our awesome community
- Installation instructions